files/helm/reloader/common.yaml

```diff
@@ -0,0 +1,133 @@
+global:
```

Comment:
delete this file. its not a good idea to add the default value

Reply:
Okey